lib/tasks/retag_editions.rake

```diff
@@ -0,0 +1,12 @@
+namespace :medical_alerts do
```

Comment:
I think it would be good for this to match the filename. Being extra picky, should we call this `medical_safety_alerts` to match the model name exactly?